lgc/elfLinker/ElfLinker.cpp

```diff
@@ -341,37 +351,35 @@ bool ElfLinkerImpl::link(raw_pwrite_stream &outStream) {
   // 2: symbol table
   // 3: .text
   // 4: .note
+  // 5: .rel.text
   m_outputSections.push_back(OutputSection(this, "", ELF::SHT_NULL));
   m_outputSections.push_back(OutputSection(this, ".strtab", ELF::SHT_STRTAB));
   m_outputSections.push_back(OutputSection(this, ".symtab", ELF::SHT_SYMTAB));
   unsigned textSectionIdx = m_outputSections.size();
+  (void)(textSectionIdx); // Used in an assert later on.
```

Comment:
Style nit: I would put this next to the assert that uses it, so later on if someone wants to remove the assert they will hopefully remove the (void) thing as well, and then notice that they get an unused variable warning.

Reply:
done.